src/main/java/javax/jmdns/impl/HostInfo.java

```diff
@@ -40,6 +40,8 @@ public class HostInfo implements DNSStatefulObject {
 
     private final HostInfoState _state;
 
+    private final static int    _labelLengthLimit = 0x3F;
```

Comment:
Is this a constraint defined in the mDNS spec?

Reply:
It's a constraint of https://www.ietf.org/rfc/rfc1035.txt, section 2.3.4

Reply:
Perfect, thanks.